### ISSUE TITLE
Redundant enable/disable autocorrection

### ISSUE THREAD

**Issue by marcandre** [rubocop-hq/rubocop#8118]
```
# rubocop:enable NotActually/Needed,IAm/Needed

# gets corrected to

# rubocop:enableIAm/Needed
```
(Note: no space after "," => no space after "enable")

**Comment by vitaly-pushkar** [comment]
I can't reproduce it. Also, I see a specific spec for it that makes sure it doesn't happen: https://github.com/rubocop-hq/rubocop/blob/master/spec/rubocop/cop/lint/redundant_cop_enable_directive_spec.rb#L134

Can you help me to reproduce it?

**Comment by rrosenblum** [comment]
I've been looking at something similar recently. The areas that I have been looking at are `--disable-uncorrectable` and `Style/DoubleCopDisableDirective`. Disable uncorrectable will produce a mix of inline comment comment disable and range comment disable even when multiple cops could fit on a single disable line. 

`Style/DoubleCopDisableDirective` assumes that all extra disable comments are invalid and removes them rather than merging them into a single comment. 

I'm having some luck with the fixes, but I am still running into some trouble with double disables from inline and range comments. I'm not sure that we have enough info at the time of disabling to fully handle this how I would like it to work.

**Comment by marcandre** [comment]
> Can you help me reproduce it?

There's [already a pull request](rubocop --auto-gen-config && rubocop -a) to fix it; I imagine that the tests include it reproduce it? If needed I encountered that on RuboCop's own source:

```sh
git fetch https://github.com/marcandre/rubocop.git
git checkout fbd49af8423e67d00628b2c0ddc35d6dd26ea087
bundle exec rubocop --auto-gen-config && rubocop -a
```

Check the last hunk on `task/cops_documentation.rake`.

**Comment by vitaly-pushkar** [comment]
Thank you @marcandre, my comment is obviously obsolete now. @lxxxvi did a good job there!

Also, I would love to dedicate some time to this project and am looking for simple issues to start with. I will look around some more, but if you have something in mind - let me know 😊 
Thank you

**Comment by marcandre** [comment]
@vitaly-pushkar There Depending on your interests and skills, there's a lot to be done. #7868 should be merged soon, and all autocorrecting cops we have could be refactored to use the new API; you can start on that now if you are familiar enough with git (the `autocorrect` branch is still forced pushed a lot). Otherwise check the issues marked "Help wanted" and/or "Good first issue". I'll create one right now
